pkg/plugins/manager/loader/loader.go

```diff
@@ -369,12 +370,12 @@ func configureAppChildPlugin(parent *plugins.Plugin, child *plugins.Plugin) {
 	if parent.IsCorePlugin() {
 		child.Module = util.JoinURLFragments("app/plugins/app/"+parent.ID, appSubPath) + "/module"
 	} else {
-		child.Module = util.JoinURLFragments("plugins/"+parent.ID, appSubPath) + "/module"
+		child.Module = util.JoinURLFragments(setting.AppSubUrl+"/public/plugins/"+parent.ID, appSubPath) + "/module.js"
 	}
 }
 
 func defaultLogoPath(pluginType plugins.Type) string {
-	return "public/img/icn-" + string(pluginType) + ".svg"
+	return "/public/img/icn-" + string(pluginType) + ".svg"
```

Comment:
does this work with appSubPath?

Reply:
Nope. Thanks for spotting this. Should be sorted now.